datadog-monitors-monitor-handler/src/datadog_monitors_monitor/handlers.py

```diff
@@ -27,6 +29,8 @@
 from datadog_api_client.v1.model.monitor_formula_and_function_events_data_source import (
     MonitorFormulaAndFunctionEventsDataSource as ApiMonitorMonitorFormulaAndFunctionEventsDataSource,
 )
+from datadog_api_client.v1.model.on_missing_data_option import OnMissingDataOption
+from datadog_api_client.v1.model.monitor_options_notification_presets import MonitorOptionsNotificationPresets
```

Comment:
Lets import these as Api<ModelName> so we can more easily differentiate between client models vs cfn.